src/scene/materials/standard-material.js

```diff
@@ -1075,6 +1077,7 @@ function _defineMaterialProps() {
     _defineFlag('emissiveTint', false);
     _defineFlag('fastTbn', false);
     _defineFlag('useMetalness', false);
+    _defineFlag('useSpecularF0', false);
```

Comment:
I think we should try and come up with some easier to understand name .. maybe just `useSpecular` or `useSpecularTint` .. as F0 does not mean much to most users.

Reply:
useMetalnessSpecular!